gradle.properties

```diff
@@ -1 +1 @@
-version=2.1.0-SNAPSHOT
+version=2.2.0-SNAPSHOT
```

Comment:
Never alter version in a pull request. It's up to the maintainer to decide when to do that and what the new version should be.